src/plugins/flexibleLayout/components/resizeHandle.vue

```diff
@@ -0,0 +1,80 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2018, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+<template>
+    <div class="c-fl-frame__resize-handle"
+         :class="[orientation]">
+    </div>
+</template>
+
+<script>
+export default {
+    props: ['orientation', 'index'],
+    data() {
+        return {
+            initialPos: 0
+        }
+    },
+    methods: {
+        mousedown(event) {
+            event.preventDefault();
+
+            this.$emit('mousedown', this.index);
+
+            document.body.addEventListener('mousemove', this.mousemove);
+            document.body.addEventListener('mouseup', this.mouseup);
+        },
+        mousemove(event) {
+            event.preventDefault();
+
+            let delta = this.getMousePosition(event) - this.getElSizeFromRect(this.$el);
+            this.$emit('mousemove', this.index, delta, event);
+        },
+        mouseup(event) {
+            this.$emit('mouseup', event);
+
+            document.body.removeEventListener('mousemove', this.mousemove);
+            document.body.removeEventListener('mouseup', this.mouseup);
+        },
+        getMousePosition(event) {
+            if (this.orientation === 'horizontal') {
+                return event.clientX;
+            } else {
+                return event.clientY;
+            }
+        },
+        getElSizeFromRect(el) {
+            if (this.orientation === 'horizontal') {
+                return el.getBoundingClientRect().x;
+            } else {
+                return el.getBoundingClientRect().y;
+            }
+        },
+    },
+    mounted() {
+        this.$el.addEventListener('mousedown', this.mousedown);
```

Comment:
bind these event handlers in the template so you don't need to bother with lifecycle.